src/main/java/Event.java

```diff
@@ -0,0 +1,36 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * An Event class that represents a Task that occurs on a day.
+ * It has a Task Description, as well as a Event date.
+ */
+public class Event extends Task {
+
+    protected LocalDate at;
```

Comment:
I think you can afford to use "private" instead of "protected" since there is not subclass for Event.